projects/tonpools/index.js

```diff
@@ -0,0 +1,11 @@
+const { call } = require("../helper/chain/ton");
+
+module.exports = {
+  timetravel: false,
+  ton: {
+    tvl: async () => {
+      const result = await call({ target: "EQA7y9QkiP4xtX_BhOpY4xgVlLM7LPcYUA4QhBHhFZeL4fTa", abi: "getActiveContest"});
```

Comment:
hi, why is this value higher than amount on ton in the contract?

Reply:
sorry, we can count only the value of tokens on the contract. I thought when user joins the lottery, the TON is deposited into this contract, is that assumption wrong?

Reply:
Yes, the user tokens are deposited in the contract only. Also, when a winner is decided the winnings are added to the contract for user to claim whenever they want. Both the winnings and user tokens deposited contribute to the total tokens in the contract.

Reply:
I have updated the code now to just pick the contract's token balance, I hope this should work.